app/assets/src/styles/_header.scss

```diff
@@ -268,22 +268,27 @@ a {
   }
 }
 
+.instruction-heading {
```

Comment:
Please move the css for the new component to its own file, i.e. `cli_user_instructions_modal.scss`